compiler/rustc_codegen_cranelift/build_system/tests.rs

```diff
@@ -88,7 +89,6 @@ const BASE_SYSROOT_SUITE: &[TestCase] = &[
     ),
     TestCase::build_bin_and_run("aot.float-minmax-pass", "example/float-minmax-pass.rs", &[]),
     TestCase::build_bin_and_run("aot.issue-72793", "example/issue-72793.rs", &[]),
-    TestCase::build_bin("aot.issue-59326", "example/issue-59326.rs"),
```

Comment:
Hey @bjorn3: I removed this test since it no longer is expected to pass (well, it fails with a deny lint, I guess I could allow it too).

If you're happy with me removing it tho, should I also remove `fn build_bin`? Or leave it as `allow(unused)` since this is the only used callsite?

Reply:
Actually, let me just restore the test and allow the lint.